lib/embulk/input/service/jql_service.rb

```diff
@@ -205,16 +201,36 @@ def extract_value(record, name)
           when NOT_PROPERTY_COLUMN
             record[NOT_PROPERTY_COLUMN]
           when "time"
-            # convert from ms -> second
             if record["time"].present?
-              time = record["time"] / 1000
-              adjust_timezone(time)
+              value = record["time"]
+              if value > 0
+                time = record["time"] / 1000
+                adjust_timezone(time)
+              else
+                value
```

Comment:
Out of curiosity, are there cases where record's time is negative?

Reply:
I think no but there are some case the value is 0. 
And adjust_timezone(0) return negative result.

Reply:
I see, good to know.